cmd/gaia/app/app.go

```diff
@@ -170,7 +170,27 @@ func NewGaiaApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest b
 	)
 	app.SetInitChainer(app.initChainer)
 	app.SetBeginBlocker(app.BeginBlocker)
-	app.SetAnteHandler(auth.NewAnteHandler(app.accountKeeper, app.feeCollectionKeeper))
+
+	authAnteHandler := auth.NewAnteHandler(app.accountKeeper, app.feeCollectionKeeper)
+
+	filterAnteHandler := func(ctx sdk.Context, tx sdk.Tx, simulate bool) (newCtx Context, result Result, abort bool) {
+		newCtx, result, abort = authAnteHandler(ctx, tx, simulate)
+		if abort == true {
+			return
+		}
+
+		for _, msg := range tx.GetMsgs() {
+			switch msg.(type) {
+			case bank.MsgMultiSend:
```

Comment:
Can we add all the other send cases here?  Then, we should delete SendEnabled as a parameter... since it's confusing to have to set it to true, for MsgMultiSend to go through... and even though that param is set in genesis, it's actually disabled... so all around confusing to have that around.

Reply:
Yeah, we should replace the SendEnabled with a proper circuit breaker in the future anyways.

Reply:
hmm, that's a good reason to keep the SendEnabled parameter as is... since the circuit breaker should ideally work through the param system.  I'll just add the other msg here.

Reply:
If we're replacing genesis parameters (not sure, I missed the discussion) we will need to filter `SetWithdrawAddress` messages here too.